### DISCUSSION TITLE
Borderless applications (disable title bar and borders)

### DISCUSSION THREAD

**Discussion by Walhaell** [WindowTop/WindowTop-App#358] [Ideas]
Good, I am delighted with your application, however, I downloaded it thinking that I could remove title bar and bords of the applications, in principle it seemed that with CROP IT could, however when entering this mode the mouse buttons (right, wheel and so on) do not work. In fact I don't know if it can be done but a simulation of what I'm looking for is something like this.

A main screen with 100% opacity and then a small one with no borders or title bar at the bottom right to be able to access when I want and need.

Again thank you very much for the software

![image](https://github.com/WindowTop/WindowTop-App/assets/169605477/0466ef4c-e5f9-430b-80ad-e361c148aac9)

**Comment by gileli121** [comment]
Hello, you are talking about the "INTERACT" mode that is available when the crop or pip is enabled 
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/fe9fc0b8-83bb-4527-8841-a249de544f81)
Please make sure that this button is toggled.

The feature works well with programs such as Edge, Chrome, Firefox, Spotify, Postman, Word, etc.
However, some apps do not work well with this feature because it is very complicated and unlikely to ever improve.

If it does not work well with this game or app, I am afraid I can't help with that.

**Reply by Walhaell** [reply]
Hi!, 

This is how I have tested if it works or not, with the interact of the pip (the button is toggled), the case is that the right click of the mouse does not work unless I have activated the option “Show original windows”, and then the left click moves me the window if I drag the cursor. Of course what I really want is to have a window of a windows application without title bar, this really what it does is to make a preview in real time of an already opened application. Another thing that I have tried is to put in minimum opacity to the original window and to be able to do what I say but the sensitivity of the game camera becomes very unstable and besides the handicap that I can not move the camera with the left click also bothers me ....

**Reply by gileli121** [reply]
" the left click moves me the window if I drag the cursor"
About this part, please disable the "Enable smart drag" option here
![image](https://github.com/WindowTop/WindowTop-App/assets/17680514/ec8b3b10-a0e5-426e-b5ec-9093a1e80a3d)

The idea is that when you drag a text, it will not drag the window, but will try to select the text instead.
But it does not work well with every app (like games, for example)

I hope that it helps
